designs/extensible-cli-and-scaffolding-plugins-phase-1.md

```diff
@@ -62,7 +62,7 @@ project versions (via `SupportedProjectVersions()`).
 Example `PROJECT` file:
 
 ```yaml
-version: "3-alpha"
+version: "2"
```

Comment:
??

Reply:
@DirectXMan12 
I understand that the version here will still be 2 since we have not broken changes in the layout. However, kb tag will need to be bumped to 3.0.0. Am I right?

Reply:
There was pushback against upgrading to "3-alpha". Regardless of whether we bump I figured it was better to make the change in a follow-up.

Reply:
that's ok, but I think if we're gonna make a field mandatory that's a change in version.